create_heatmaps/base.py

```diff
@@ -18,35 +20,43 @@ def __init__(self, config, index):
 
         # file paths
         self.metadata_path = config["metadata_paths"][index]
-        self.lcdata_path = config["lcdata_paths"][index]
-        self.output_path = config["heatmaps_path"]
+        self.lcdata_path   = config["lcdata_paths"][index]
+        self.output_path   = config["heatmaps_path"]
         self.finished_filenames_path = os.path.join(self.output_path, "finished_filenames.csv")
 
         # heatmap parameters / metadata
         self.wavelength_bins = config["num_wavelength_bins"]
-        self.mjd_bins = config["num_mjd_bins"]
-        self.has_peakmjd = config.get("has_peakmjd", True)
+        self.mjd_bins        = config["num_mjd_bins"]
+        self.has_peakmjd     = config.get("has_peakmjd", True)
 
         # heatmap labeling
         self.categorical = config["categorical"]
-        self.types = config["types"]
-        print(self.types)
+        self.types       = config["types"]
+        logging.info(f"List of types: {self.types}")
         self.sn_type_id_to_name = config["sn_type_id_to_name"] # SNANA type ID to type name (i.e. 42 -> SNII)
         self.type_to_int_label = {type_str: 1 if type_str == "SNIa" or type_str == "Ia" else 0 for type_str in self.types} if not self.categorical else {v:i for i,v in enumerate(sorted(self.types))} # int label for classification
-        print(f"type to int label: {self.type_to_int_label}")
+
+        logging.info(f"type to int label: {self.type_to_int_label}")
 
         # restricting number of heatmaps that are made
         self.ids_path = config.get("ids_path", None)
 
+        # get sim fraction (Mar 1 2024)
+        self.prescale_heatmap = config['prescale_heatmap']
+        ps = self.prescale_heatmap
```

Comment:
is this temp variable needed?

Reply:
technically the temp 'ps' variable is not needed, at least for now. There are two long-term maintenance reasons for using these temp variables:
1) the actual 'ps' that is used could become more complex later, so it only needs to be updated in one place, rather than the two places used below;
2) code below is a little more readable with simpler name, and we might add more code later using ps.